sdk/spring/azure-spring-data-cosmos/src/test/java/com/azure/spring/data/cosmos/core/CosmosTemplateIT.java

```diff
@@ -967,6 +967,7 @@ public void createWithAutoscale() throws ClassNotFoundException {
         assertNotNull(throughput);
         assertEquals(Integer.parseInt(TestConstants.AUTOSCALE_MAX_THROUGHPUT),
             throughput.getProperties().getAutoscaleMaxThroughput());
+        collectionManager.deleteContainer(autoScaleSampleInfo);
```

Comment:
should this be in a try finally block - just in case somehow we got some failures